util.sh

```diff
@@ -8,13 +8,13 @@
 #   deploy: Bundles up required files for the Python app, deploys to Phantom VM, and invokes the Phantom app compilation script.
 #   deploy-bypass: Bypass the password prompts using the PHANTOM_VM_PASSWORD environment variable.
 #   ssh: SSH into your Phantom VM.
-#   open-web: Open your default web browser at `https://<phantom-vm-ip:9999`.
+#   open-web: Open your default web browser at `https://<phantom-vm-ip:9999>`.
 
 set -eo pipefail
 
 make_tar() {
   echo "Tarring the ball..."
-  pushd .. && tar -cvf phcode42v2/phcode42v2.tgz -X phcode42v2/exclude_files.txt phcode42v2/* && popd
+  tar -cvf phcode42v2.tgz -X ./exclude_files.txt -H .
```

Comment:
I found that this command was not properly excluding the files because when we use `pushd` the lines in `exclude_files` no longer match anything. The `-H` makes it so that symlinks are followed, which is need for the `code42_connector.py` link that I added at the root.